wom/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,18 @@
+<configuration>
```

Comment:
Did you mean to check this file in or was this just for testing?

Reply:
oh nm, you address this in your description. I'm not used to such well-written descriptions, I wasn't expecting that. 😛